Automattic/jetpack
# Launchpad: Add notice to wp-admin sidebar ( ON HOLD )

Related https://github.com/Automattic/wp-calypso/issues/69756

#### Changes proposed in this Pull Request:

* If a site has gone through a newsletter or link-in-bio onboarding flow and Launchpad tasks have not been completed, display a Launchpad notice in the `wp-admin` sidebar.
* Note that we already do this in the Calypso admin sidebar ( see https://github.com/Automattic/wp-calypso/pull/70325 )

#### Screenshots
![Screen Shot 2022-11-30 at 2 25 55 PM copy](https://user-images.githubusercontent.com/5414230/204921675-275c6cf8-fa33-4e8c-b20a-6e4cddec9a3d.jpg)

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






WPCOM Simple
* Create a new site by going through the new newsletter or link in bio tailored onboarding flows ( `wordpress.com/setup/newsletter/intro` or `wordpress.com/setup/link-in-bio/intro` )
* Don't complete Launchpad tasks
* Sandbox site
* `bin/jetpack-downloader test jetpack add/launchpad_notice_to_admin_sidebar`
* Navigate to `wp-admin` for the site
* Verify that the launchpad notice is displayed and redirects to Launchpad Ex. https://wordpress.com/setup/newsletter/launchpad?siteSlug=YOUR_SITE_SLUG_HERE

Atomic
* Since Launchpad isn't functioning properly on Atomic sites, as of right now, we might be unable to test the notice. We can, however, test to ensure that current JITM notices aren't broken.
* Same instructions as above, except instead of using the jetpack-downloader, follow the steps below.
* Get an Atomic site.
* Install Jetpack Beta on that site.
* Use Jetpack Beta to apply this branch on your site.
